src/explore-education-statistics-common/src/modules/table-tool/components/TimePeriodForm.tsx

```diff
@@ -77,6 +77,16 @@ const TimePeriodForm = (props: Props & InjectedWizardProps) => {
     return matchingOption ? matchingOption.label : '';
   };
 
+  const getDisplayTimePeriod = (
+    startValue: string,
+    endValue: string,
+  ): string => {
+    if (startValue === endValue) {
+      return getOptionLabel(startValue);
+    }
+    return `${getOptionLabel(startValue)} to ${getOptionLabel(endValue)}`;
```

Comment:
I think this change might break the following UI test so worth double checking it 🙂 
`publish_release_and_amend_2.robot:192`
`user checks previous table tool step contains    4    Time period    2012/13 to 2012/13`

Reply:
Good spot, I checked the table tool ones but missed this. Fixed now.